### PR TITLE
Attempt to make the Definition docs a bit better

### PR DESCRIPTION
**Pull request by philsherry**
Gave these docs an overhaul:

- Definition of Ready
- Definition of Done

The first one was just a legacy list of _things_ that never really made sense.

### CODE REVIEW

File: docs/definition-of-ready.md [hunk 1]
```diff
@@ -1,24 +1,63 @@
...
+- use non-breaking spaces and hyphens to stop unwanted word wrapping
+- your design must be mobile-friendly (remember a zoomed in desktop screen will effectively become a mobile viewport)
+- do not convey meaning by colour alone
+- there should reasonable distance between related elements, rather than remote from each other
+
+### Design for assistive technology
+
+- add hidden context for screen readers, where necessary
+- notify screen readers when onscreen content or layout changes, using relevant ARIA roles
+- provide a warning when an action takes the user to a different tab or window
```
Comment (adamliptrot-oc) [inline docs/definition-of-ready.md:54]: "_provide a warning when an action **will take** the user ..._"

File: docs/definition-of-ready.md [hunk 2]
```diff
@@ -1,24 +1,63 @@
...
+- all form elements have accessible names
```
Comment (adamliptrot-oc) [inline docs/definition-of-ready.md:55]: all **interactive** elements instead of just **form**?

**Review by adamliptrot-oc — CHANGES_REQUESTED** [review]

File: docs/definition-of-ready.md [hunk 3]
```diff
@@ -1,24 +1,63 @@
...
-- simple, easy to understand content that meets the content style guide
-- how will this look on mobile (remember a zoomed in desktop screen will effectively become a mobile viewport)
-- how this will work without javascript
-- if the journey is more than a few screens and whether we allow the user to save and come back later
-- whether people using assistive technology might need extra time to complete tasks
+## Be inclusive
+
+To make sure accessibility is part of the story, include elements of the following:
+
+- [established patterns](#established-patterns)
```
[the remark this answers is not in the document]
Comment (philsherry) [reply docs/definition-of-ready.md:11]: These sections should certainly have more things in them, but this was all I thought of at the time. I think it's valid, but feel free to add further items.

File: docs/definition-of-ready.md [hunk 4]
```diff
@@ -1,24 +1,63 @@
...
+- reading order should make sense when there are differences between what is visible and what is read out loud
+- write page titles that are the same as the `h1` heading (but be aware of the [Personally identifiable information](https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/#personally-identifiable-information) caveat)
+
+### Visual design
+
+- colour contrast ratios should be fully accessible
+- use non-breaking spaces and hyphens to stop unwanted word wrapping
+- your design must be mobile-friendly (remember a zoomed in desktop screen will effectively become a mobile viewport)
+- do not convey meaning by colour alone
+- there should reasonable distance between related elements, rather than remote from each other
```
[the remark this answers is not in the document]
Comment (philsherry) [reply docs/definition-of-ready.md:48]: We discussed this on Slack at the time—this was _your own wording_. 😆

**Comment by philsherry** [comment]
This has got a bit out of control due to that unverified commit at the top of the chain, so I'll close this before it gets any worse and breaks GitHub.

**Pull request closed without merging**